aws/ecs-service/outputs.tf

```
@@ -0,0 +1,4 @@
+output "task_execution_role_arn" {
+  description = "Task execution role arn"
+  value       = "aws_iam_role.ecs_tasks_execution_role.arn"
```

```suggestion
  value       = aws_iam_role.ecs_tasks_execution_role.arn
```
Last bit; this shouldn't be a string